src/atoms/button/component.tsx

```diff
@@ -79,19 +80,24 @@ export interface IButtonProps extends IComponentProps<IButtonTheme> {
   onClicked?(): void;
```

Comment:
i think you need to read through this one more: https://dev.to/bytebodger/default-props-in-react-ts-part-deux-2ic3

Many of these props are required in the current interface but we need to make them optional and use the `< AllPropsRequired> as in the linked blog post so that the errors dont appear for props that have a default value (which we can set for all)

Reply:
I would expect to see the `setDefaults` defined somewhere as in the linked blog post